src/components/icon/icon.stories.js

```diff
@@ -19,16 +19,8 @@ export default {
 };
 
 const commonKnobs = () => {
-  const tooltipMessage = text("tooltipMessage", "");
   return {
-    tooltipMessage,
```

Comment:
I think this knob should stay, as we decided to keep the Tooltip in Icon right?

Reply:
yep good spot